_content/tour/ind/channels.article

```diff
@@ -36,166 +34,166 @@ channels.
-- [[https://www.ardanlabs.com/blog/2014/02/the-nature-of-channels-in-go.html][The Nature Of Channels In Go]] - William Kennedy    
-- [[http://matt-welsh.blogspot.com/2010/07/retrospective-on-seda.html][A Retrospective on SEDA]] - Matt Welsh    
-- [[https://www.youtube.com/watch?v=KBZlN0izeiY][Understanding Channels]] - Kavya Joshi    
+- [[https://www.ardanlabs.com/blog/2017/10/the-behavior-of-channels.html][The Behavior Of Channels]] - William Kennedy
+- [[https://golang.org/ref/mem#tmp_7][Channel Communication]]
+- [[http://blog.golang.org/share-memory-by-communicating][Share Memory By Communicating]] - Andrew Gerrand
+- [[https://www.ardanlabs.com/blog/2014/02/the-nature-of-channels-in-go.html][The Nature Of Channels In Go]] - William Kennedy
+- [[http://matt-welsh.blogspot.com/2010/07/retrospective-on-seda.html][A Retrospective on SEDA]] - Matt Welsh
+- [[https://www.youtube.com/watch?v=KBZlN0izeiY][Understanding Channels]] - Kavya Joshi
 
 ** Buffer Bloat - 2011
 
-Be careful about using large buffers with the idea of reducing latency.
+Berhati-hatilah dalam menggunakan buffer besar dengan tujuan mengurangi latensi.
 
-- Large buffers prevent timely notification of back pressure.
-- They defeat your ability to reduce back pressure in a timely matter.
-- They can increase latency not reduce it.
-- Use buffered channels to provide a way of maintaining continuity.
-- Don't use them just for performance.
-- Use them to handle well defined bursts of data.
-- Use them to deal with speed of light issues between handoffs.
+- Buffer besar mencegah pemberitahuan tekanan balik yang tepat waktu.
+- Mereka menghancurkan kemampuan Anda untuk mengurangi tekanan balik dengan cepat.
+- Mereka dapat meningkatkan latensi bukan menguranginya.
+- Gunakan buffered channels untuk memberikan cara mempertahankan kontinuitas.
+- Jangan gunakan mereka hanya untuk masalah kinerja.
+- Gunakan mereka untuk menangani burst data yang terdefinisi dengan baik.
+- Gunakan mereka untuk menangani masalah kecepatan cahaya antara serah terima.
 
-*Videos*
+*Video*
 
-- [[https://www.youtube.com/watch?v=qbIozKVz73g][Bufferbloat: Dark Buffers in the Internet]]  
-- [[http://www.bufferbloat.net/projects/cerowrt/wiki/Bloat-videos][Buffer Bloat Videos]]  
+- [[https://www.youtube.com/watch?v=qbIozKVz73g][Bufferbloat: Dark Buffers in the Internet]]
+- [[http://www.bufferbloat.net/projects/cerowrt/wiki/Bloat-videos][Buffer Bloat Videos]]
 
-* Exercises
+* Latihan
 
-Use the template as a starting point to complete the exercises. A possible solution is provided.
+Gunakan templat sebagai titik awal untuk menyelesaikan latihan. Solusi telah disediakan.
 
-** Exercise 1
+** Latihan 1
 
-Write a program where two goroutines pass an integer back and forth ten times.
-Display when each goroutine receives the integer. Increment the integer with each
-pass. Once the integer equals ten, terminate the program cleanly.
+Tulis program di mana dua goroutine saling melewatkan integer sepuluh kali.
```

Comment:
this one isnt't quite right

suggestion:

Tulis program di mana dua goroutine saling mengoper sebuah integer sepuluh kali.

Reply:
ack